app/models/package.scala

```diff
@@ -93,7 +93,7 @@ package object models {
           val updatedJsArray = valueToRemoveFrom.value.slice(0, index) ++ valueToRemoveFrom.value.slice(index + 1, valueToRemoveFrom.value.size)
           JsSuccess(JsArray(updatedJsArray))
         case valueToRemoveFrom: JsArray => JsError(s"array index out of bounds: $index, $valueToRemoveFrom")
-        case _ => JsError(s"cannot set an index on $valueToRemoveFrom")
+        case null => JsError(s"cannot set an index on $valueToRemoveFrom")
```

Comment:
The wildcard _ catches all cases that aren't matched by the previous patterns.By changing to case null, you're only catching the null case,but there are many other possible non-array values that could be passed,  is there a specific reason to only catch null values?